FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithRetrofitService.kt

```diff
@@ -3,26 +3,43 @@ package com.flagsmith.internal;
 import android.content.Context
 import android.util.Log
 import com.flagsmith.FlagsmithCacheConfig
+import com.flagsmith.entities.FeatureStatePutBody
 import com.flagsmith.entities.Flag
 import com.flagsmith.entities.IdentityFlagsAndTraits
 import com.flagsmith.entities.TraitWithIdentity
+import okhttp3.Cache
 import okhttp3.Interceptor
 import okhttp3.OkHttpClient
 import retrofit2.*
 import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.http.Body
 import retrofit2.http.GET
+import retrofit2.http.Header
 import retrofit2.http.POST
+import retrofit2.http.PUT
+import retrofit2.http.Path
 import retrofit2.http.Query
 
-interface FlagsmithRetrofitService {
+internal interface FlagsmithRetrofitService {
```

Comment:
As per previous comment, would this constitute a breaking change? If so, can we implement it separately?

Reply:
This is intentional so that we don't expose the 'write' API to users.

Reply:
I've exposed this again now